packages/components/src/button/index.native.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * External dependencies
+ */
+import { TouchableOpacity } from 'react-native';
+
+/**
+ * WordPress dependencies
+ */
+import { forwardRef } from '@wordpress/element';
+
+export function Button( props ) {
+	const { children, onClick, 'aria-label': ariaLabel } = props;	
+	return (
+		<TouchableOpacity
+			accessible={ true }
+			accessibilityLabel={ ariaLabel }
+			onPress={ onClick }
+		>
+			{ children }
+		</TouchableOpacity>
+	);
+}
+
+export default forwardRef( Button );
```

Comment:
`forwardRef` won't work this way, besides it is not used inside the component. I would simply remove it from native implementation.